src/Microsoft.DotNet.MSBuildSdkResolver/MSBuildSdkResolver.cs

```diff
@@ -53,15 +53,19 @@ public override SdkResult Resolve(SdkReference sdkReference, SdkResolverContext
                 netcoreSdkVersion = new DirectoryInfo(netcoreSdkDir).Name;;
             }
 
-            if (!string.IsNullOrEmpty(sdkReference.MinimumVersion) && 
-                SemanticVersion.Parse(netcoreSdkVersion) < SemanticVersion.Parse(sdkReference.MinimumVersion))
+            FXVersion netCoreSdkFXVersion;
+            FXVersion minimumFXVersion;
+            if (!FXVersion.TryParse(netcoreSdkVersion, out netCoreSdkFXVersion) ||
+                !FXVersion.TryParse(sdkReference.MinimumVersion, out minimumFXVersion) ||
+                netCoreSdkFXVersion < minimumFXVersion)
```

Comment:
If netcoreSdkVersion is null here, we should consider it to be larger than any minimum. It means we got msbuildSdksDir from the environment, which is an override for forcing the Sdks to use (for testing). I don't think this override should require picking a version (though you can with a separate environment variable).